kANe/v2p1/07_Upanayana.md

```diff
@@ -12,26 +12,35 @@ This word literally means 'leading or taking near'. But the important question i
+A few words about the origin and development of this most important saṁskāra would not be out of place here. Comparison with the ancient Zoroastrian scriptures ( vide S. B. E. vol. V. pp. 285-290 about the sacred girdle and shirt) and the modern practices among the Parsis of India tend to show that Upanayana goes back to an Indo-Iranian origin. But that subject is outside the scope of this work. Confining ourselves to Indian Literature, we find that already in the Ṛg. X. 109.5 the word 'brahmacāri' occurs 'oh gods! he ( Bṛhaspati), all pervading one, moves as a brahmacārī pervading all (sacrifices); he is only one part of the gods ( i. e. of sacrifices ); Bṛhaspati secured by that (i. e. by his service to the gods ) a wife (me who am named) Juhū, who was (formerly ) taken by Soma'. [617] The word 'upanayana' [618] can be derived and explained in two ways: (1) taking (the boy) near the ācārya, (2) that rite by which the boy is taken to the ācārya. The first sense appears to have been the original one and when an extensive ritual came to be associated with upanayana the second came to be the sense of the word. Such an ancient work as the Āp. Dh. S. I.1.1. 19 says that upanayana is a saṁskāra (purificatory rite ) laid down by revelation for him who seeks learning (i. e. it accepts the second explanation) or this sūtra may mean 'it is a saṁskāra brought about by imparting the śruti ( viz. Gāyatrī mantra ) to him who seeks learning'.  This would mean that upanayana principally is gāyatryupadeśa ( tho imparting of the sacred Gāyatri mantra). This appears to be suggestod by the Vedic passage quoted above ( p.154 f. n. 356) 'he created the brāhmaṇa with Gāyatrī, the kṣatriya with Triṣṭubh, the vaiśya with Jagatī', and by Kātyāyana Śrauta 'one should initiate the brāhmaṇa with Gāyatrī'. Jaimini [619] also (in VI. 1.35) propounds the view that upanayana is a saṁskāra and has a seen result viz. thereby the boy is brought near the teacher for learning Veda.
+
+[617]: ब्रह्मचारी चरति वेविषद् विषः स देवानां भवत्येकमङ्गम्। तेन जायामन्वषिन्दद्  बृहस्पति: सोमेन नीतां जुह्वं न देवा:। ऋ. X. 109.5 =अथर्व V.17.5. The reference to Soma recalls Ṛg. X.85.45 'सोमो ददद्र्न्धर्वाय.' Every girl was supposed to have been under the protection of Soma, Gandharva and Agni before her marriage with a human bridegroom. 
```

Comment:
Should be [^617]. Similarly other footnotes.

Reply:
sorry about that. missed it. Now corrected.